README.md

```diff
@@ -1 +1,180 @@
-# math-magician
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+ <!-- LOGO -->
+
+  <img src="/public/logo512.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+<!-- MAIN HEADING -->
+
+  <h3><b>React App</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- INTRO -->
+# 📖 React App <a name="about-project"></a>
+
+> This is a new React App.
+
+## 🛠 Built With <a name="built-with"></a>
+1. React
+2. CSS
+3. JavaScript
+4. Webpack
+### Tech Stack <a name="tech-stack"></a>
+
+> Tech Stack to be updated soon
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    <li><a href="https://reactjs.org/">JavaScript</a></li>
+    <li><a href="https://reactjs.org/">Webpack</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> - Display animated react logo
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+LIVE DEMO
+
+> -
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy up and running, follow these steps.
+
+<!-- SETUP -->
+### Setup
+
+To setup this project, run this command:
+
+```npm run test
+```
+### Prerequisites
+
+1. A Browser (Preferably Google Chrome)
+2. A Code Editor
+3. Internet Connection
+4. Git
+
+<!-- INSTALL -->
+### Install
+
+Install this project with Iroko.
+
+### Usage
+
+To run the project, execute the following command:
+
+### Run tests
+> npm run test
+### Deployment
+
+You can deploy this project using:
+>1.  GitHub Pages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Anyars Yussif**
+
+- GitHub: [@anyars-encarta](https://github.com/anyars-encarta)
+- Twitter: [@anyarsencarta](https://twitter.com/anyarsencarta)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/anyars-yussif-1a179769/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **Add a calculator feature**
```

Comment:
- [ ] I'll suggest you remove these as they are placeholder comments which help you fill out your readme. :small_airplane: 

- - Screenshot:
![Selection_008](https://github.com/anyars-encarta/math-magician/assets/107632780/e22315c4-5ef0-4bf8-ab52-f236d4c60e20)